101-data-factory-blob-2-sql/azuredeploy.json

```diff
@@ -0,0 +1,194 @@
+{
+  "contentVersion": "1.0.0.0",
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "parameters": {
+    "dataFactoryName": {
+      "type": "string",
+      "metadata": {
+        "description": "Data Factory Account name"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage Account name"
+      }
+    },
+    "storageAccountKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Storage Account Key"
+      }
+    },
+    "sourceContainer": {
+      "type": "string",
+      "metadata": {
+        "description": "Source file container name"
+      }
+    },
+    "sqlServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure SQL Database Server name"
+      }
+    },
+    "sqlServerUserName": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure SQL Database Server user name"
+      }
+    },
+    "sqlServerPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Azure SQL Database Server password"
+      }
+    },
+    "targetDatabaseName": {
+      "type": "string",
+      "metadata": {
+        "description": "Taget Azure SQL Database name"
+      }
+    },    
+    "targetSQLTable": {
+      "type": "string",
+      "metadata": {
+        "description": "Target tabl in Azure SQL Database"
+      }
+    },
+    "location": {
```

Comment:
What is limiting more locations? Is it the Microsoft.DataFactory/datafactories RP?